### ISSUE TITLE
Ownership type filter options showing  ids (?) instead of names

### ISSUE THREAD

**Issue by ostermanj** [codefordc/housing-insights#551]
![image](https://user-images.githubusercontent.com/13643664/30492620-1e601d86-9a0f-11e7-90cf-4392e6c45765.png)

Use to be text of the ownership type, now showing numbers instead.

**Comment by NealHumphrey** [comment]
@ostermanj Yep, this was due to the latest batch of data that we got from Urban institute, where they had done their export wrong and just gave us codes. I've gotten the new data from Peter but I have not yet loaded it into the database. In the new batch of data I also got some better cleaned up ownership names that are shorter/easier to understand.
